forge-gui/res/cardsfolder/upcoming/epic_confrontation.txt

```diff
@@ -0,0 +1,6 @@
+Name:Epic Confrontation
```

Comment:
This card already exists in DTK so this script isn't needed

Reply:
oh, good catch !